src/web/browsers/browsers-config.helper.ts

```diff
@@ -11,6 +11,9 @@ const getDefaultBrowsersConfigs = (config): any => {
     chromeOptions: {
       args: [],
       prefs: {
+        intl: {
+          accept_languages: config.browserLanguage,
```

Comment:
Add a default language, so we don't loose the compatibility

Reply:
This should be described in MIGRATION rules. We shouldn't add defaults here